api/util/errorLog.js

```diff
@@ -9,13 +9,13 @@ const ErrorLog = require('../models/ErrorLog');
  * @param {string} req.body.apiEndpoint - The location of the error
```

Comment:
change the variable names of these comments, e.g.
```suggestion
 * @param {string} errorToAdd.apiEndpoint - The location of the error
```